velox/exec/tests/utils/CMakeLists.txt

```diff
@@ -32,22 +32,23 @@ add_library(
   PortUtil.cpp)
 
 target_link_libraries(
-  velox_exec_test_lib
-  velox_vector_test_lib
-  velox_temp_path
+  velox_aggregates
```

Comment:
This is not correct. The first one is the target name - and must be `velox_exec_test_lib`. See also the Cmake documentation for `target_link_libraries`.

Reply:
Moved `velox_exec_test_lib` to beginning of the list